batchflow/models/validator/validator.py

```diff
@@ -0,0 +1,172 @@
+""" API for models and validator """
+
+import warnings
+
+import yaml
+
+from ..metrics import ClassificationMetrics, SegmentationMetricsByPixels, \
+                      SegmentationMetricsByInstances, RegressionMetrics
+
+METRICS = dict(
```

Comment:
there is `batchflow.pipeline.METRICS`. You can use it (and even move to batchflow namepsace)